### ISSUE TITLE
Bold Words Often Become Different Characters

### ISSUE THREAD

**Issue by pan-asian-wok** [ItMarki/jyunsaikaai#4]
When I bold words, bold words often become different characters. I don't think it's a program-related issue, since I have tried it on multiple programs with the same results. Below is an example:

![image](https://github.com/ItMarki/jyunsaikaai/assets/45274474/37f1c49a-cde3-4fa7-811c-820d2e439264)

![image](https://github.com/ItMarki/jyunsaikaai/assets/45274474/5e1f5b20-03fb-4d14-9c85-1c978750c90c)

**Comment by maxmellen** [comment]
I can confirm that I’m running into the same issue on macOS!

**Comment by Tenun-negi** [comment]
![Screenshot_20230610-103150_MT_Manager](https://github.com/ItMarki/jyunsaikaai/assets/93485080/eb0e108a-c984-49a6-ac7c-10fb0a230831)
![Screenshot_20230610-103145_MT_Manager](https://github.com/ItMarki/jyunsaikaai/assets/93485080/bf633174-477b-412d-a676-d87eb1d15829)
Same.

**Comment by jkwchui** [comment]
I took a look inside the fonts, and my sense is that this cannot be easily fixed.

The fonts internally use unicode codepoints as the reference, but the "Bold" variant has different glyph content (overall 34 *more* glyphs), and skip/slippages happen on both Bold and Regular fonts.

![image](https://github.com/ItMarki/jyunsaikaai/assets/775675/a586d773-2115-486a-9139-eae01ffc0b8c)

In the above image, for example, we see that the Regular is missing a glyph for the codepoint `2F14`, but Bold filled in `2F14` as the name for the next glyph.  This off-by-one error carries through until it encounter the next slippage.  From the font `ttf` itself, we only get to see the paths and with the font containing odd glyphs (like the irregular hiragana), it is likely difficult to reconcile the codepoint with the appearance without manual labor.

----

I note that the bold / light were implemented as "shells" (and anti-shells) over the Regular variant:

![image](https://github.com/ItMarki/jyunsaikaai/assets/775675/f95eebf5-b790-4d76-bdc6-56fac0a05628)

Given this, I think it is actually faster to re-generate the bold from Regular, than trying to fix the irregularly offset codepoints.  If the author-maintainer @ItMarki is open to this, we can look into how to make this happen.
